### DISCUSSION TITLE
using SDR++ from the command line to record a wav

### DISCUSSION THREAD

**Discussion by prismspecs** [AlexandreRouma/SDRPlusPlus#1390] [General]
Hey all,
Thanks for the great software. I have compiled it on my Pi 5 64bit. I would like to record FM transmissions at specific times throughout the day. Is there a simple way to do this via the command line with this program? It seems from the documentation that it may involve setting up a SpyServer, but what if I want to do it on-device? I see there is a server option, but then is there an API to tell it what frequency, when to start and stop recording, etc?
Thank you for your time

**Comment by AlexandreRouma** [comment]
There is not. This is a GUI program, not a command line program, and there is no plans to make it one.

**Reply by prismspecs** [reply]
Thanks for the response!
